src/test/resources/application-test.yml

```diff
@@ -0,0 +1,53 @@
+#
```

Comment:
The only other thing I can think to try so that you wouldn't have to add this file to the repo would be to set a default value for `brapi.server.default-url` in `application.yml`

Reply:
If you did that wouldn't you be limited to only changing the value in application.yml because it seems like it would still have trouble finding the env?

Reply:
No, you can always override a value in `application.yml` by specifying the same hierarchy in `application-prod/test/etc.yml`

Reply:
For github actions though it doesn't seem like you'd be able to otherwise it would be working now right?

Reply:
Added default url to pom and removed application-test. Looks like it worked.